core/store/migrate/migrations/0169_log_poller_pruning.sql

```diff
@@ -0,0 +1,10 @@
+-- +goose Up
+-- +goose StatementBegin
+ALTER TABLE evm_log_poller_filters ADD COLUMN retention BIGINT DEFAULT 0;
```

Comment:
0 retention presumably means "never delete" right?

Reply:
Yes, this is mentioned in the PR description and a couple comments.  0 = permanent retention.  It makes the DELETE query slightly more complex , but I think it makes for a cleaner interface than NULL or MAXINT.  With NULL, you'd have to allocate memory for a uint64 each time and pass a pointer to it, just to set a retention which is annoying.  And with all the conversion between time.Duration BIGINT, etc. I don't have as much trust in the portability of MAXINT and seems messier.

Reply:
I should add a line to `doc.go` about this feature too, actually.